prog/vm/nexus.rb

```diff
@@ -490,6 +490,14 @@ def before_run
       hop_update_firewall_rules
     end
 
+    when_checkup_set? do
+      hop_unavailable if !available?
+    rescue Sshable::SshError
+      # Host is down, which will be handled by HostNexus
+      # No need to create a page for each Vm separately.
+      decr_checkup
```

Comment:
Should we hop to the `unavailable` label and decide not to create a page there? This way, both the operator and the customer can see that the VM is unavailable.

Reply:
makes sense, changed it.